### ISSUE TITLE
Album artist info from Spotify

### ISSUE THREAD

**Issue by flickur** [web-scrobbler/web-scrobbler#2946]
Hi,

Could we get the album artist also automatically from Spotify so you don't need to always fill it up manually? How it works now is that it just uses the song artist as album artist if the "album artist" field is empty but that's not correct in many cases e.g. when there's a compilation album that features songs from multiple artist like [this](https://open.spotify.com/album/3YyW1hbmpNZ7X2LiuLNMyo?si=LHoVml6KThyoPmwf3NJHdg), and results album scrobbles to be spread under multiple different album artists. 

One thing that would help a lot is if you could just set an album artist (and artist) for an album once and the scrobbler would apply the correction to all tracks on that album. I'm aware this has been suggested but just mentioning it since it would be super useful feature.

**Comment by Aspiring-Writer** [comment]
I agree, this would be nice. It's annoying to have to manually edit songs on Spotify because Web Scrobbler will put all the featured artists in the artist field with the main artist. The mobile scrobbler I use, called Pano Scrobbler, automatically omits multiple artists and only includes the first one. If there was some way that WS could automatically exclude ", ...." that would be a huge time-saver.

**Comment by amd64fox** [comment]
I confirm that it is impossible to scrobble from spotify because of this problem, I have been writing about this to developers for a long time, but it has not been fixed yet, it's a pity.
